### PR TITLE
DVRescue MediaArea related pages

### PR DESCRIPTION
**Pull request by JeromeMartinez**
Just a place holder for MediaArea work on DVRescue, see [DVRescue annoncement from MIPoPS](https://www.mipops.org/dvrescue/) for more info.
@dericed please review text in `src/DVRescueBundle/Resources/views/Default/index.html.twig` in order to be sure there is no mistake about who is responsible for what.

### CODE REVIEW

**Comment by dericed** [comment]
@libbyhopfauf, @ablwr, can you review https://github.com/MediaArea/MediaArea-Website/pull/307/files#diff-841fd1965479310e5cedf15fb321da39. This adds dvrescue to the mediaarea site and links back to mipops.

File: src/DVRescueBundle/Resources/views/Default/license.html.twig [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{% extends 'DVRescueBundle::base.html.twig' %}
```
Comment (ablwr) [inline src/DVRescueBundle/Resources/views/Default/license.html.twig:1]: I would like to see the license change settled and represented, as per here: https://github.com/mipops/dvrescue/issues/10, but that could be a subsequent PR.

**Comment by ablwr** [comment]
Content-wise this is okay (although I did make a comment about changing the License), but Travis seems unhappy about something about the DownloadController (or maybe tests need updating to match the new pages).

File: src/DVRescueBundle/Resources/views/Default/license.html.twig (hunk 1 above)
Comment (JeromeMartinez) [reply src/DVRescueBundle/Resources/views/Default/license.html.twig:1]: I show the current license in this PR so I suggest that I merge with it and change in another PR as soon as another license is chosen.

**Comment by JeromeMartinez** [comment]
> (or maybe tests need updating to match the new pages).

I inadvertently put Linux page tests, but Linux downloads are not yet there, removed (they'll be back when I do a Linux release).

Now just waiting for official decision about license.

**Comment by ablwr** [comment]
License issue not a blocker for this PR in my opinion

**Review by ablwr — APPROVED** [review]

**Review by dericed — APPROVED** [review]

**Comment by ablwr** [comment]
I noticed the homepage lists some but not all projects. A bit tangential to this PR, but just a reminder that you may want to change/update that page as well. And you very well may not!

**Comment by ablwr** [comment]
Depending on your data model, you could consider consolidating the front-page hype for the 3 metaedits, add in RAWcooked, and swap DV Analyzer for DVRescue, for example

**Comment by JeromeMartinez** [comment]
True, MediaArea 2 latest projects are not listed on main page, and they could be..
I am reluctant to consolidate the MetaEdit projects are they are (currently) independent.
BTW, https://MediaArea.net/DVRescue is online.

**Comment by ablwr** [comment]
thank you!

**Pull request merged**